index.html

```diff
@@ -2612,6 +2617,96 @@ <h3 id="applying">
       </section>
       <section>
         <h3>
+          <code title="">category</code> member
+        </h3>
+        <p>
+          The <dfn id="member-category"><code>category</code>
+          member</dfn> describes the expected application category (or 
+          categories) to which the web application belongs. This is a hint 
+          to catalogs or stores of web applications under which category 
+          (or categories) to list this web application.
+        </p>
+        <p>
+          The <code>category</code> member is only meant as a hint to catalogs 
+          or stores listing web applications and it is expected that these 
+          will make a best effort to find an appropriate category under which 
+          to list the web application.
+        </p>
+        <p>
+          The <dfn>steps for processing the <code>category</code>
+          member</dfn> are given by the following algorithm. The algorithm
+          takes a <var>manifest</var> as an argument. This algorithm returns a
+          string or <code>undefined</code>. The string MAY include a comma 
```

Comment:
Yeah, I think we should use an array. That's what they are there for :) Hopefully we didn't screw that up elsewhere. I can't remember :(